public/pages/Home/components/PostFilter.tsx

```diff
@@ -76,7 +87,8 @@ export const PostFilter = (props: PostFilterProps) => {
   const options: OptionItem[] = []
 
   if (fider.session.isAuthenticated) {
-    options.push({ value: true, label: i18n._({ id: "home.postfilter.option.myvotes", message: "My Votes" }), type: "myVotes" })
+    options.push({ value: true, label: i18n._({ id: "home.postfilter.option.myvotes", message: "My votes" }), type: "myVotes" })
+    options.push({ value: true, label: i18n._({ id: "home.postfilter.option.myposts", message: "My posts" }), type: "myPosts" })
```

Comment:
```suggestion
    options.push({ value: true, label: i18n._({ id: "home.postfilter.option.myvotes", message: "My Votes" }), type: "myVotes" })
    options.push({ value: true, label: i18n._({ id: "home.postfilter.option.myposts", message: "My Posts" }), type: "myPosts" })
```

(You'll need to change the strings that have been put into client.json too)